components/image-builder-api/typescript/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.1.5",
   "license": "UNLICENSED",
   "scripts": {
-    "build": "tsc && cp -f src/*.js src/*d.ts lib"
+    "build": "cp -f src/*.js src/*d.ts lib && tsc"
```

Comment:
Doesn't this fundamentally change the semantics? To build it, we'd first build and then copy the files over but now we're copying before we build. Or are these semantically equivalent because `tsc` and the copy are unrelated to each other?